.travis.yml

```diff
@@ -10,9 +10,11 @@ language: cpp
 # To experiment with Python 3, comment out Python 2.7 and uncomment one of the Python 3 versions.
 env:
   global:
-    - PYTHON_VERSION="2.7.15"
+    # - PYTHON_VERSION="2.7.15"
     # - PYTHON_VERSION="3.6.7"
-    # - PYTHON_VERSION="3.7.1"
+    - PYTHON_VERSION="3.7.1"
```

Comment:
should this be 3.7.4 now?
and how about just removing those commented out lines above, I don't think they're helpful here

Reply:
Pyenv Python is different than Travis CI.  The latter supports Python 3.7.4 and 3.8.0 but I do not believe the former does yet.

Reply:
left at 3.7.1, removed the extra lines